controllers/v1/library/institutionalController.js

```diff
@@ -0,0 +1,255 @@
+                    errorObject: error
+                });
+            }
+        })
+    }
+
+    /**
+    * @api {get} /assessment/api/v1/institutional/metaForm/:solutionId Institutional library Meta Form
+    * @apiVersion 1.0.0
+    * @apiName Institutional Solution Meta Form
+    * @apiGroup Institutional Library
+    * @apiHeader {String} X-authenticated-user-token Authenticity token
+    * @apiSampleRequest /assessment/api/v1/institutional/metaForm/5ed5ec4dd2afa80d0f616460
+    * @apiUse successBody
+    * @apiUse errorBody
+    * @apiParamExample {json} Response:
+    * "result": [
+        {
+            "field": "name",
+            "label": "Title",
+            "value": "",
+            "visible": true,
+            "editable": true,
+            "validation": {
+                "required": true
+            },
+            "input": "text"
+        }
+    ]
+    */
+
+     /**
+   * Institutional library meta form.
+   * @method
+   * @name metaForm
+   * @param {Object} req -request Data.
+   * @returns {JSON} - Institutional library meta form.
+   */
+
+   async metaForm(req) {
+
+    return new Promise(async (resolve, reject) => {
+
+        try {
+
+            let institutionalMetaForm = 
+            await institutionalHelper.metaForm(
+                req.params._id
+            );
+
+            return resolve(institutionalMetaForm);
+
+        } catch (error) {
+            return reject({
+                status: error.status || httpStatusCode.internal_server_error.status,
+                message: error.message || httpStatusCode.internal_server_error.message,
+                errorObject: error
+            });
+        }
+    });
+   }
+
+  /**
+     * @api {post} /assessment/api/v1/library/institutional/make/:librarySolutionId make institutional solution
+     * @apiVersion 1.0.0
+     * @apiName Make solution from template
+     * @apiGroup Institutional Library
+     * @apiParamExample {json} Request-Body:
+     * @apiSampleRequest /assessment/api/v1/library/institutional/make/5ed5ec4dd2afa80d0f616460
+     * @apiUse successBody
+     * @apiUse errorBody
+     * @apiParamExample {json} Response:
+     * {
+     * "message": "Successfully created solution",
+     * "status": 200,
+     * "result": {
+     * "program": {
+     * "_id": "5ed8f45af779fd187322acfb",
+     * "name": "Aman",
+     * "externalId": "AFRICA-ME-TEST-FRAMEWORK-TEMPLATE-BASE-SOLUTION-2020-06-02 19:04:04-2020-06-04 18:47:14"
+     * },
+     * "solution": {
+     * "_id": "5ed8f45af779fd187322acfc",
+     * "name": "Aman",
+     * "externalId": "AFRICA-ME-TEST-FRAMEWORK-TEMPLATE-BASE-SOLUTION-2020-06-02 19:04:04-2020-06-04 18:47:14"
+     * }
+     * }
+     * }
+     */
+     
+    /**
+    * Make solution from institutional library template
+    * @method
+    * @name template
+    * @param {Object} req -request Data.
+    * @returns {JSON} - Make solutions from institutional library template.
+    */
+
+   async make(req) {
```

Comment:
@aman-tunerlabs - As discussed **make** API need not be here, can be in observations and assessment controllers itself